src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -454,17 +454,19 @@ private void checkTranslationKeys(ResourceBundle bundle) {
     private void checkFilesForConsistencyRegardingTheirKeys(Map<File, Set<String>> fileKeys,
                                                             Set<String> keysThatMustExist) {
         for (Entry<File, Set<String>> fileKey : fileKeys.entrySet()) {
-            final MessageDispatcher dispatcher = getMessageDispatcher();
-            final String path = fileKey.getKey().getPath();
-            dispatcher.fireFileStarted(path);
             final Set<String> currentFileKeys = fileKey.getValue();
             final Set<String> missingKeys = keysThatMustExist.stream()
                 .filter(key -> !currentFileKeys.contains(key)).collect(Collectors.toSet());
-            for (Object key : missingKeys) {
-                log(1, MSG_KEY, key);
+            if (!missingKeys.isEmpty()) {
```

Comment:
Added empty check to ensure we don't report a file is being started if we have no violations to report.